src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -71,6 +71,16 @@ EXTERN_C REDHAWK_API void __cdecl RhFlushProcessWriteBuffers()
     PalFlushProcessWriteBuffers();
 }
 
+// Bulk zero memory operation for large memory blocks
+EXTERN_C REDHAWK_API void __cdecl RhZeroMemory(void* pMemStart, size_t cbMemSize)
```

Comment:
Does this *need* to be a new runtime service? Could the class library just p/invoke into `memset`, same as we already do for `memmove`?